ui/src/containers/SampleViewContainer.jsx

```diff
@@ -115,22 +121,28 @@ class SampleViewContainer extends Component {
         <Row className="gx-3 mt-2 pt-1">
           <Col sm={2} xxl={1} className={styles.controllers}>
             <DefaultErrorBoundary>
-              <div className={motorInputStyles.container}>
-                <label className={motorInputStyles.label} htmlFor="PhaseInput">
-                  Phase Control
-                </label>
-                <PhaseInput />
-              </div>
-
-              <div className={motorInputStyles.container}>
-                <label
-                  className={motorInputStyles.label}
-                  htmlFor="ApertureInput"
-                >
-                  Beam size
-                </label>
-                <ApertureInput />
-              </div>
+              {phase_control !== undefined && (
```

Comment:
Late to the party, so no big deal (more for future reference) but two very small stylistic points here:

- camel case: `phaseControl` instead of `phase_control`
- `phase_control !== undefined &&` should be equivalent to `phase_control &&` since `phase_control` is an object if I'm not mistaken.